scripts/nn/examples/Example-MNIST_2NN_Leaky_ReLu_Softmax.dml

```diff
@@ -20,12 +20,15 @@
 #-------------------------------------------------------------
 
 /*
- * The MNIST Data can be downloaded as follows:
- * mkdir -p data/mnist/
- * cd data/mnist/
- * curl -O https://pjreddie.com/media/files/mnist_train.csv
- * curl -O https://pjreddie.com/media/files/mnist_test.csv
- */
+* This Example trains a feed forward neural network with one input layer, two hidden affine layers (200 neurons) with
+* leaky relu activations and one affine output layer with a softmax activation
```

Comment:
These changes are only telling us what is added.

But, not WHY these changes are added. If you have contacted any SystemDS developer offline, consider providing
the summary of the discussion here.